### PR TITLE
Add support for unregistering a login callback

### PR DESCRIPTION
**Pull request by tmtrademarked**
Thanks for proposing a pull request.

To help us review the request, please complete the following:
- [ X ] sign contributor license agreement: https://developers.facebook.com/opensource/cla
- [ X ] submit against our `:dev` branch, not `master`.
- [ X ] describe the change (for example, what happens before the change, and after the change)

======

This PR adds an API to allow a caller to unregister their callback. This is essential in order to avoid memory leaks if the callback is retained. It's pretty trivial to produce a memory leak today if the callback provided to the SDK contains a reference to an Activity.

With this PR, it's simple to unregister. This allows the user to avoid leaking whatever the implementation of their callback is.

### CODE REVIEW

File: facebook/src/main/java/com/facebook/login/widget/LoginButton.java [hunk 1]
```diff
@@ -456,6 +456,16 @@ public void registerCallback(
...
     }
 
+    /**
+     * Unregisters a login callback to the given callback manager.
+     *
+     * @param callbackManager The callback manager that will encapsulate the callback.
+     */
+    public void unregisterCallback(
+            final CallbackManager callbackManager) {
+        getLoginManager().unregisterCallback(callbackManager);
```
Comment (zaren678) [inline facebook/src/main/java/com/facebook/login/widget/LoginButton.java]: Looks like the spacing is off here a little.

Comment (tmtrademarked) [inline facebook/src/main/java/com/facebook/login/widget/LoginButton.java]: Adjusted - thanks for looking!

**Comment by premnirmal** [comment]
🎉

**Comment by tmtrademarked** [comment]
Awesome - thanks for reviewing, @zaren678 ! What's the process for getting this merged from here? (Just curious)

**Pull request merged**